test/commitment/fri.cpp

```diff
@@ -95,7 +95,7 @@ ACTOR_THREAD_TEST_CASE(fri_basic_test) {
     constexpr static const std::size_t lambda = 40;
     constexpr static const std::size_t batches_num = 1;
 
-    typedef zk::commitments::fri<FieldType, merkle_hash_type, transcript_hash_type, lambda, m, batches_num> fri_type;
+    typedef zk::commitments::fri<FieldType, merkle_hash_type, transcript_hash_type, lambda, m, true> fri_type;
```

Comment:
Does that mean we have batch size equal 1?

Reply:
Nope, Elena changes the commitments totally. Now this "true" means "use grinding", and there are not batches number any more as far as I understand.